src/virt.c

```diff
@@ -2059,7 +2059,9 @@ static int start_event_loop(virt_notif_thread_t *thread_data) {
   if (pthread_create(&thread_data->event_loop_tid, NULL, event_loop_worker,
                      thread_data)) {
     ERROR(PLUGIN_NAME " plugin: failed event loop thread creation");
+    virt_notif_thread_set_active(thread_data, 0);
```

Comment:
good catch. I wonder if we should just initialize notif_thread before attempting to connect.

Reply:
Wouldn't it be equivalent to only move the above instruction `virt_notif_thread_set_active(thread_data, 1);` just after `if (pthread_create(...)) {...}` statement ? The value of `thread_data->is_active` must be `false` before this statement because function `start_event_loop` is called after `virt_notif_thread_init`.

Reply:
Thanks for your review!

I think it is not equivalent. Started thread will check `is_active` and it has to be `true` on its start.

Reply:
I just noticed BTW in `lv_shutdown` that line `DEBUG(PLUGIN_NAME " plugin: stopping event loop");` should be inside the next `if (!persistent_notification)` statement.

Reply:
>event loop....

The same is about `starting event loop`.
I think, if change this, it is better to put these debug messages into `start_event_loop() / stop_event_loop()` then.

Reply:
Sorry @rpv-tomsk , I didn't see your answers to my above comments and I removed one of them to start a code review. Anyway, I agree with both your answers. Debug messages can be moved to mentioned functions.